examples/celery-rabbitmq/rabbitmq-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1beta3
+kind: Service
+metadata:
+  labels:
+    name: rabbitmq-service
+  name: rabbitmq-service
+spec:
+  port: 5672
+  ports:
+  - port: 5672
+    protocol: TCP
+    targetPort: 5672
+  selector:
+    app: taskQueue
```

Comment:
Which pods is this service supposed to target? It will target all pods with the "app: taskQueue" label, which includes flower, rabbitmq, and celery. Add "name: rabbitmq" if it should just target the rabbitmq pods.